script.js

```diff
@@ -0,0 +1,196 @@
+        <button class="see">${project.button}</button>
+        </div>
+      </div>
+      </div>
+    `;
+
+    projectsContainer.innerHTML += projectHTML;
+  } else if (i === 1) {
+    const project1HTML = `
+      <div class="grid-projects">
+      <div class="see-proj">
+        <div class="projects-background" id="whitebg">  
+        <button class="sees-project" id="see-proj-btn">${project.button}</button>
+                    
+        </div>
+      </div>  
+      `;
+    const projectElement = document.getElementById(`project-${i}`);
+    projectElement.addEventListener('mouseenter', () => {
+      const imageElement = document.getElementById(`project-img-${i}`);
+      const buttonElement = document.getElementById(`see-proj-btn-${i}`);
+      imageElement.classList.add('hover');
+      buttonElement.classList.add('hover');
+    });
+
+    projectElement.addEventListener('mouseleave', () => {
+      const imageElement = document.getElementById(`project-img-${i}`);
+      const buttonElement = document.getElementById(`see-proj-btn-${i}`);
+      imageElement.classList.remove('hover');
+      buttonElement.classList.remove('hover');
+    });
+
+    projectsGrid.innerHTML += project1HTML;
+  } else {
+    const projectHTML = `
+      <div class="see-proj">
+        <div class="projects-background" id="bg${i}"> 
+          <div class="background">
+            <h2 class="projects-heading">${project.title}</h2>
+            ${project.title1 ? `<h2 class="Desktop-heading">${project.title1}</h2>` : ''}
+            <p class="para">${project.description}</p>
+            <div class="project-btn">
+              <ul>${project.tags.map((tag) => `<li class="projects-btn">${tag}</li>`).join('')}</ul>
+            </div>
+          </div>  
+        </div>
+        <button class="see-project">${project.button}</button>
+      </div>
+    `;
+
+    projectsGrid.innerHTML += projectHTML;
+  }
+}
+
+function valid() {
+  const email = document.getElementById('email').value;
+  const validatedEmail = email.toLowerCase();
+  if (validatedEmail === email) {
+    document.querySelector('.error').textContent = '';
+    return true;
+  }
+  document.querySelector('.error').textContent = 'Email address should always be in Lowercase.';
+  return false;
+}
+
+const form = document.querySelector('form');
+form.addEventListener('submit', (event) => {
+  if (valid() === false) {
+    event.preventDefault();
+  }
+});
+
+const nameInput = document.getElementsByName('user-name');
+const emailInput = document.getElementsByName('e-mail');
+const messageInput = document.getElementsByName('message');
+
+let formData = {
+  nameInputValue: '',
+  emailInputValue: '',
+  messageInputValue: '',
+};
+
+if (localStorage.getItem('form-data')) {
+  formData = JSON.parse(localStorage.getItem('form-data'));
+  nameInput[0].value = formData.nameInputValue;
+  emailInput[0].value = formData.emailInputValue;
+  messageInput[0].value = formData.messageInputValue;
+}
+
```

Comment:
Good job working on the preserve data in the browser 😄, but you need to fix some parts 👍.

- [ ] Kindly make sure when the user changes the content of any input field, the data is saved to the local storage as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=When%20the%20user%20changes%20the%20content%20of%20any%20input%20field%2C%20the%20data%20is%20saved%20to%20the%20local%20storage.).  
- [ ] Also make sure when the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=When%20the%20user%20loads%20the%20page%2C%20if%20there%20is%20any%20data%20in%20the%20local%20storage%20the%20input%20fields%20are%20pre%2Dfilled%20with%20this%20data.). Kindly fix it 😃.

![Capture](https://user-images.githubusercontent.com/57408419/232252177-9ebf218c-7a31-410f-92c9-7034d149517a.PNG)